instance/models/server.py

```diff
@@ -384,7 +395,7 @@ def start(self,
         else:
             self.openstack_id = os_server.id
             self.logger.info('Server got assigned OpenStack id %s', self.openstack_id)
-            # Persist OpenStack ID
+            # Persist OpenStack ID ans public IP
```

Comment:
This change is wrong anyway and should be reverted.  (I first tried to store the public IP immediately after VM creation, but it's not yet available at that time.)